client/packages/android/capacitor.config.ts

```diff
@@ -0,0 +1,31 @@
+import { CapacitorConfig } from '@capacitor/cli';
+
+// This file is used to generate packages/android/app/src/main/assets/capacitor.config.json
```

Comment:
Another file that cause me some headaches to get right, the hostname setting was a bit hard to figure out